### PR TITLE
Add more details in readme for macos builds.

### PR DESCRIPTION
**Pull request by kgpai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,11 +81,30 @@ git submodule update --init --recursive
 
 ### Setting up on macOS
 
-See [scripts/setup-macos.sh](scripts/setup-macos.sh)
+Once you have checked out Velox, On an Intel MacOS machine you can setup and then build like so :
+
+```shell
+$ ./scripts/setup-macos.sh 
+$ make
```
[the remark this answers is not in the document]
Comment (kgpai) [reply README.md:88]: We actually do that in section below called Building Velox, make is just set to compile sources.

**Review by mbasmanova — APPROVED** [review]
Thanks.

**Review by bikramSingh91 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -81,11 +81,30 @@ git submodule update --init --recursive
 
 ### Setting up on macOS
 
-See [scripts/setup-macos.sh](scripts/setup-macos.sh)
+Once you have checked out Velox, On an Intel MacOS machine you can setup and then build like so :
```
Comment (majetideepak) [inline README.md:84]: nit: `on`. 
nit: `build like so :` (remove space)

File: README.md [hunk 3]
```diff
@@ -81,11 +81,30 @@ git submodule update --init --recursive
...
+```
+
+On an M1 MacOS machine you can build like so:
+
+```shell
+$ CPU_TARGET="arm64" ./scripts/setup-macos.sh
+$ CPU_TARGET="arm64" make
+```
+
+You can also produce intel binaries on an M1, just `CPU_TARGET="sse"` for the above.
```
Comment (majetideepak) [inline README.md:98]: nit: `just`->`use`

File: README.md [hunk 4]
```diff
@@ -81,11 +81,30 @@ git submodule update --init --recursive
...
+
+```shell
+$ ./scripts/setup-macos.sh 
+$ make
+```
+
+On an M1 MacOS machine you can build like so:
+
+```shell
+$ CPU_TARGET="arm64" ./scripts/setup-macos.sh
```
Comment (majetideepak) [inline README.md:94]: We should probably rename `CPU_TARGET` to `VELOX_CPU_TARGET`?

**Review by majetideepak — COMMENTED** [review]
Thanks, @kgpai. Lazy review. Some more thoughts.

**Pull request closed without merging**